python/CONTRIBUTING.rst

```diff
@@ -0,0 +1,131 @@
+.. highlight:: shell
```

Comment:
As above - can we copy in a file or generate the `.rst` from the `.md` rather than manually duplicating content? (Will Python packages take `.md` these days for docs?)